### ISSUE TITLE
[Joss review] Improve statement of need and edit paper for length/clarity

### ISSUE THREAD

**Issue by rmsare** [martibosch/detectree#3]
The paper is well written and clearly states the intended audience for this package. 

Some suggestions to make it more accessible: 

- [x] The statement of need could be made more specific and highlighted in the first paragraph.

I think it is currently *"The target audience of DetecTree is researchers and practitioners in GIS that are interested in two-dimensional aspects of trees, such as their proportional abundance and spatial distribution throughout a region of study."* This highlights what you can measure using detectree, but it would nice to extend this to mention research problems that use these measurements.

- [x] The statement of need should be added to the top of the README and docs site.

 - [ ] In the final paragraph (or introduction), a sentence or two could be added with a specific example of analysis using detectree.

What kinds of problems have you solved using maps of urban tree extent? I'm sure there are many nice applications, like estimating quality-of-life at a neighborhood level or work in sustainable development. 

- [x] The detailed description of the core method (from "The aim ..." to Step 4) might be better placed in its own "Methodology" sub-section before "Availability". 

Readers may appreciate seeing a high-level overview of what dectectree does, who might want to use it, and what kind of API is available before diving into the details. This is clearly laid out in paragraphs 1, 3, and 4 of the paper.

[JOSS review](https://github.com/openjournals/joss-reviews/issues/2172)

**Comment by rmsare** [comment]
Clicking around, I see you've produced a tree canopy map of Lausanne using detectree. This would be a nice example to cite in the paper!

**Comment by rmsare** [comment]
I think the urban heat island example you give in your talk is a good one, too. Estimating the tree cover of road pixels in a separate dataset using detectree output would make a nice step-by-step example for the docs.

**Comment by martibosch** [comment]
I have moved the statement of need as the second paragraph and I have added a sentence mentioning a possible research problem, i.e., evaluation of the provision of ecosystem services. I have also added such statement in the README.

The methodology has now a dedicated section, which should further address #5.

Regarding the example application, the tree canopy map of Lausanne is part of our study of urban heat islands, nevertheless we do not have any proper citation for it yet - we hope to have it by the end of the month. In the meantime, I have added a sentence in the "Availability" section mentioning the Lausanne tree canopy repository.
